entity_view_builder.module

```diff
@@ -2,25 +2,51 @@
 
 /**
  * @file
- * Implements hooks.
+ * Contains hook implementations.
  */
 
+use Drupal\entity_view_builder\EntityViewBuilderConfigManager;
+
 /**
  * Implements hook_entity_type_alter().
  *
- * Replace the default view builder controller with our form that will
- * delegate all methods to alter plugins with the specific entity type.
+ * Replace the default view builder controller with custom view builders
+ * that will delegate the build method.
  */
 function entity_view_builder_entity_type_alter(array &$entity_types) {
-  // Add the custom view builder handler for node entity types.
-  if (isset($entity_types['node'])) {
-    $node_entity_type = $entity_types['node'];
-    $node_entity_type->setHandlerClass('view_builder', 'Drupal\entity_view_builder\ViewBuilder\NodeViewBuilder');
+  $container = \Drupal::getContainer();
+  $config_manager = $container->get('config.manager.entity_view_builder');
+
+  // Get configuration from all .delegation_config.yml files.
+  $config = $config_manager->getConfiguration();
+  $view_builders = [];
+
+  // Change from per-module array association to per-entity type.
+  foreach ($config as $module) {
+    foreach ($module as $view_builder) {
+      $view_builders[$view_builder['entity_type_id']][] = [
+        'class' => $view_builder['class'],
+        'priority' => $view_builder['priority'],
+      ];
+    }
+
   }
 
-  // Add the custom view builder handler for paragraph entity types.
-  if (isset($entity_types['paragraph'])) {
-    $paragraph_entity_type = $entity_types['paragraph'];
-    $paragraph_entity_type->setHandlerClass('view_builder', 'Drupal\entity_view_builder\ViewBuilder\ParagraphViewBuilder');
+  foreach ($view_builders as $entity_type_id => &$list) {
+    // Sort the Entity View Builders, decreasingly,  based on the priority;
+    usort($list, function($a, $b) {
+      if ($a['priority'] == $b['priority']) {
+        return 0;
+      }
+
+      return ($a['priority'] < $b['priority']) ? 1 : -1;
+    });
+
```

Comment:
There is no need to check for this if, because you did that in `EntityViewBuilderConfigManager->sanitizeList()` .